Library/ViewModels/PledgeViewModel.swift

```diff
@@ -44,13 +44,7 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
           .map { Format.date(secondsInUTC: $0, template: "MMMMyyyy", timeZone: UTCTimeZone) } ?? ""
         let shipping = (
           "Brooklyn",
-          Format.attributedCurrency(
```

Comment:
If you take a look at https://github.com/kickstarter/ios-oss/pull/682/files, I've already removed this call to `Format.attributedCurrency` in favor of sending the `Project` directly through to the cell, where we can pipe it into a `ViewModel` that does this formatting. How should we resolve these two approaches?

One thing we could do is leave the `shippingValue(for: project)` function at the bottom of this file, but remove it from anywhere that it's called - that way we can simply move it over to the `PledgeShippingLocationCellViewModel` for usage once that refactor happens, and we shouldn't have any merge conflicts. What do you think?